### DISCUSSION TITLE
Nordics RSE Unconference as a way to catch up with other RSEs

### DISCUSSION THREAD

**Discussion by manuGil** [TU-Delft-DCC/TU-Delft-DCC.github.io#9] [Archive]
They are planning an online unconference on **June 29–30.** The program will consist of your contributions and we encourage you to submit a short abstract for a discussion topic, talk, demonstration, or any other type of program you would like to run.

This event is not only limited to the Nordics and not only limited to participants who are or identify as RSEs, but anybody interested in the interface of research, software, and data is welcome.

**Shall we join?**

**June 29-30, submit talks technology-related topics** : https://nordic-rse.org/events/2021-online-unconference/

**Comment by jurra** [comment]
@manuGil it looks like a great opportunity. Any ideas for topics you would like to bring?
I am curious about IaaS (infrastructure as service) for research software. How do other colleagues around the world setup computational environments for researchers. Do they have a standard procedure or infrastructure in place for that? How do they deal with storage for instance and also archiving of data. Might be interesting also to ask questions about how do they support researchers in their own context.

**Reply by manuGil** [reply]
@niketagrawal  also suggested to join the NL RSE Meetup on the same day: https://nl-rse.org/pages/meetups.html
